packages/docusaurus-plugin-content-blog/src/validation.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as Joi from '@hapi/joi';
+
+export const DefaultOptions = {
+  feedOptions: {},
+  beforeDefaultRehypePlugins: [],
+  beforeDefaultRemarkPlugins: [],
```

Comment:
Hey, are these options a typo?

That looks like, please confirm so that I can remove in my current pr

Reply:
ah no it seems used in practice

Reply:
I copied it from docs.https://v2.docusaurus.io/docs/using-plugins/#docusaurusplugin-content-blog

```
        beforeDefaultRemarkPlugins: [],
        beforeDefaultRehypePlugins: [],
        feedOptions: {
          type: '', // required. 'rss' | 'feed' | 'all'
          title: '', // default to siteConfig.title
          description: '', // default to  `${siteConfig.title} Blog`
          copyright: '',
          language: undefined, // possible values: http://www.w3.org/TR/REC-html40/struct/dirlang.html#langcodes
        },

````

Reply:
> ah no it seems used in practice

Sorry I didn't understand what u mean.

Reply:
nevermind I thought it was not used but it is